.github/CODEOWNERS

```diff
@@ -300,6 +300,8 @@
 /pkg/proto/                             @DataDog/agent-shared-components
 /pkg/proto/connectionserver/            @DataDog/universal-service-monitoring @DataDog/Networks
 /pkg/proto/datadog/languagedetection    @DataDog/processes
+/pkg/proto/datadog/process              @DataDog/processes
+/pkg/proto/datadog/trace                @DataDog/agent-apm
```

Comment:
Could you please add `@DataDog/agent-apm` as owner of `pkg/proto/pbgo/trace` ?
I noticed that this [PR](https://github.com/DataDog/datadog-agent/pull/19963) wasn't auto assigned to any team for review.

Reply:
Done